build-tools/remap-assembly-ref/remap-assembly-ref.csproj

```diff
@@ -6,15 +6,13 @@
     <OutputType>Exe</OutputType>
     <RootNamespace>remapassemblyref</RootNamespace>
     <AssemblyName>remap-assembly-ref</AssemblyName>
-    <OutputPath>..\..\bin\Build$(Configuration)</OutputPath>
+    <OutputPath>..\..\bin\Build$(Configuration)\remap-assembly-ref</OutputPath>
   </PropertyGroup>
 
   <Import Project="..\..\Configuration.props" />
 
   <ItemGroup>
-    <Reference Include="Mono.Cecil">
-      <HintPath>$(XamarinAndroidSourcePath)external\mono\sdks\out\android-bcl\monodroid_tools\Mono.Cecil.dll</HintPath>
-    </Reference>
+    <PackageReference Include="Mono.Cecil" Version="0.11.2" />
```

Comment:
Can we make the version a property in `Configuration.props`? I'm working on a PR which will add a tool that also uses Cecil, it would be good to share the package version

Reply:
Yeah, that makes sense, added. 👍